js/ui/calendar/ui.calendar.js

```diff
@@ -909,15 +966,22 @@ const Calendar = Editor.inherit({
     },
 
     _swipeEndHandler: function(e) {
+        const { viewsCount, currentDate, rtlEnabled } = this.option();
         const targetOffset = e.event.targetOffset;
         const moveOffset = !targetOffset ? 0 : targetOffset / Math.abs(targetOffset);
 
+        const isAdditionalViewActive = this._isAdditionalViewDate(currentDate);
+        const shouldDoubleOffset = viewsCount > 1 && rtlEnabled
```

Comment:
Seems like `viewsCount` cannot be `<=1` if `isAdditionalViewActive === true`, am a right?
```suggestion
        const shouldDoubleOffset = isAdditionalViewActive 
            && moveOffset === (rtlEnabled ? -1 : 1);
```

Reply:
yes, but there is also a case when multiview is enabled, main view is active and we should double

we should double when: 
 1) multiview is enabled (which means viewsCount > 1) AND
 2) (mainView is active and offset === 1) OR (additionalView is active and offset === -1)

Reply:
I didn't catch. Main view is active => `isAdditionalViewActive` = false => both branches of ternary returns false

Reply:
It was you suggestion to check isAddtitionalViewActive befare ternary :)

Reply:
condition updated